Tests/StreamChatUITests/UISDKdocumentation/UIView+Annotations.swift

```diff
@@ -0,0 +1,321 @@
+            descriptionLabel.textColor = textColor
+        }
+    }
+}
+
+private func addLine(fromPoint start: CGPoint, toPoint end: CGPoint, in container: UIView, color: UIColor) -> CGPoint {
+    let line = CAShapeLayer()
+    let linePath = UIBezierPath()
+    linePath.move(to: start)
+    linePath.addLine(to: end)
+    line.path = linePath.cgPath
+    line.strokeColor = color.cgColor
+    line.fillColor = color.cgColor
+    line.lineWidth = 0.5
+    container.layer.addSublayer(line)
+    
+    return end
+}
+
+extension String {
+    func deletingPrefix(_ prefix: String) -> String {
+        guard hasPrefix(prefix) else { return self }
+        return String(dropFirst(prefix.count))
+    }
+}
+
+public extension _ChatChannel {
+    /// Creates a new `_ChatChannel` object from the provided data.
+    static func documentationMock(
+        cid: ChannelId,
+        name: String? = "channelName",
+        imageURL: URL? = XCTestCase.TestImages.yoda.url,
+        lastMessageAt: Date? = .init(timeIntervalSince1970: 1_168_332_060),
+        createdAt: Date = .init(timeIntervalSince1970: 1_168_332_060),
+        updatedAt: Date = .init(timeIntervalSince1970: 1_168_332_060),
+        deletedAt: Date? = nil,
+        createdBy: _ChatUser<ExtraData.User>? = nil,
+        config: ChannelConfig = .mock(),
+        isFrozen: Bool = false,
+        lastActiveMembers: [_ChatChannelMember<ExtraData.User>] = [],
+        membership: _ChatChannelMember<ExtraData.User>? = nil,
+        currentlyTypingMembers: Set<_ChatChannelMember<ExtraData.User>> = [],
+        lastActiveWatchers: [_ChatUser<ExtraData.User>] = [],
+        unreadCount: ChannelUnreadCount = .noUnread,
+        watcherCount: Int = 0,
+        memberCount: Int = 2,
+        reads: [_ChatChannelRead<ExtraData>] = [],
+        extraData: ExtraData.Channel = .defaultValue,
+        latestMessages: [_ChatMessage<ExtraData>] = [],
+        muteDetails: MuteDetails? = nil
+    ) -> Self {
+        self.init(
+            cid: cid,
+            name: name,
+            imageURL: imageURL,
+            lastMessageAt: lastMessageAt,
+            createdAt: createdAt,
+            updatedAt: updatedAt,
+            deletedAt: deletedAt,
+            createdBy: createdBy,
+            config: config,
+            isFrozen: isFrozen,
+            lastActiveMembers: { lastActiveMembers },
+            membership: membership,
+            currentlyTypingMembers: currentlyTypingMembers,
+            lastActiveWatchers: { lastActiveWatchers },
+            unreadCount: { unreadCount },
+            watcherCount: watcherCount,
+            memberCount: memberCount,
+            reads: reads,
+            extraData: extraData,
+            latestMessages: { latestMessages },
+            muteDetails: { muteDetails },
+            underlyingContext: nil
+        )
+    }
+    
+    /// Creates a new `_ChatChannel` object for  from the provided data.
+    static func documentationMockDMChannel(
```

Comment:
BTW, I don't see any coding using this? 🤔

Reply:
Thanks, deleted